logging/api/logging_macros.h

```diff
@@ -76,6 +77,20 @@ static const bool _lf_log_level_is_debug = LOG_LEVEL >= LOG_LEVEL_DEBUG;
     }                                                                                                                  \
   } while (0)
 
+/**
+ * @brief A macro used to print timestamped(physical time) debug information.
+ * @ingroup API
+ *
+ * @note This macro is functionally same as @ref LF_PRINT_DEBUG but with added timestamp for physical time.
+ * This is to check physical time of debug logs. It uses @ref lf_time_physical to get the physical time.
+ */
+#define LF_TIMESTAMP_PRINT_DEBUG(format, ...)                                                                          \
+  do {                                                                                                                 \
+    if (_lf_log_level_is_debug) {                                                                                      \
+      lf_print_debug("[%ld] " format, lf_time_physical_elapsed(), ##__VA_ARGS__);                                             \
```

Comment:
Unfortunately, in C, the format directive for long long is not standardized, so `%ld` will not work on some platforms.  We have a workaround, which is to use the `PRINTF_TIME` macro.  I'm not sure whether having a macro within a macro will work, but if it does, it would need to look something like this:
```
      lf_print_debug("[" PRINTF_TIME "] " format, lf_time_physical_elapsed(), ##__VA_ARGS__);                                             \
```